backend/models/schemas.py

```
@@ -0,0 +1,15 @@
+"""
+Pydantic schemas for request and response validation.
+"""
+from pydantic import BaseModel
```

Ruff (0.11.9)

`pydantic.BaseModel` imported but unused

Remove unused import: `pydantic.BaseModel`

(F401)